app/controllers/reports_controller.rb

```diff
@@ -4,9 +4,13 @@ class ReportsController < ApplicationController
   require_permission :can_view_reports?
 
   def index
-    @org_by_county = {}
-    Organization.counties.sort_by { |county| (county.presence || "no county").downcase }.each do |county_name|
-      @org_by_county[county_name.presence || "No County"] = Organization.where(county: county_name)
+    # raise params.inspect
+    @report = params[:report]
+    if @report == "value_by_county"
+      @org_by_county = Organization.by_county_report
+    elsif @report == "total_inventory_value"
+      @category = Category.find(params[:category_id]) if params[:category_id]
```

Comment:
How about have this report set `@categories` always, so it would be `@categories = [Category.find(params[:category_id])] if params[:category_id].present?`

This would make the view a bit simpler.